src/main/java/com/atlassian/bitbucket/jenkins/internal/trigger/BitbucketWebhookMultibranchTrigger.java

```diff
@@ -35,21 +35,6 @@ public BitbucketWebhookMultibranchTrigger() {
         return (BitbucketWebhookMultibranchTrigger.DescriptorImpl) super.getDescriptor();
     }
 
-    @Override
-    public void start(MultiBranchProject<?, ?> project, boolean newInstance) {
```

Comment:
Will need to update the `BitbucketSCMSourceIT`, which relies on this method.

Reply:
Yup working on it now, thanks.

- [ ] fix `BitbucketSCMSourceIT`